### ISSUE TITLE
Search movies to rate does not work

### ISSUE THREAD

**Issue by rbestardpino** [rbestardpino/rankio#2]
When searching for movies in "/rate" throws this error and does not search

```
rate-17cb1e984f4b830d.js:1 Mixed Content: The page at 'https://rankio.bepi.tech/rate' was loaded over HTTPS, but requested an insecure resource 'http://api.themoviedb.org/3/search/movie?api_key=fc38338f58588b810badb2febb318cf1&language=en-US&query=s'. This request has been blocked; the content must be served over HTTPS.
(anonymous) @ rate-17cb1e984f4b830d.js:1
rate-17cb1e984f4b830d.js:1          Uncaught (in promise) TypeError: Failed to fetch
    at rate-17cb1e984f4b830d.js:1:9483
    at s (main-101cfeaa18eb0e64.js:1:87653)
    at Generator._invoke (main-101cfeaa18eb0e64.js:1:87441)
    at Generator.P.forEach.e.<computed> [as next] (main-101cfeaa18eb0e64.js:1:88076)
    at u (rate-17cb1e984f4b830d.js:1:8581)
    at a (rate-17cb1e984f4b830d.js:1:8784)
    at rate-17cb1e984f4b830d.js:1:8843
    at new Promise (<anonymous>)
    at rate-17cb1e984f4b830d.js:1:8724
    at h (rate-17cb1e984f4b830d.js:1:9613)
rate-17cb1e984f4b830d.js:1 Mixed Content: The page at 'https://rankio.bepi.tech/rate' was loaded over HTTPS, but requested an insecure resource 'http://api.themoviedb.org/3/search/movie?api_key=fc38338f58588b810badb2febb318cf1&language=en-US&query=sd'. This request has been blocked; the content must be served over HTTPS.
(anonymous) @ rate-17cb1e984f4b830d.js:1
rate-17cb1e984f4b830d.js:1          Uncaught (in promise) TypeError: Failed to fetch
    at rate-17cb1e984f4b830d.js:1:9483
    at s (main-101cfeaa18eb0e64.js:1:87653)
    at Generator._invoke (main-101cfeaa18eb0e64.js:1:87441)
    at Generator.P.forEach.e.<computed> [as next] (main-101cfeaa18eb0e64.js:1:88076)
    at u (rate-17cb1e984f4b830d.js:1:8581)
    at a (rate-17cb1e984f4b830d.js:1:8784)
    at rate-17cb1e984f4b830d.js:1:8843
    at new Promise (<anonymous>)
    at rate-17cb1e984f4b830d.js:1:8724
    at f (rate-17cb1e984f4b830d.js:1:9304)
```
![image](https://user-images.githubusercontent.com/49445152/154869895-e0208d69-e11e-4baa-becd-21601470288b.png)

**Comment by rbestardpino** [comment]
Try if making it an API call to /api/search movie works

**Comment by rbestardpino** [comment]
Fixed with commit 74a18da
